controllers/openstackipset_controller.go

```diff
@@ -149,7 +149,7 @@ func (r *OpenStackIPSetReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			r.Client,
 			instance.Namespace,
 			instance.Spec.Networks[0],
-			currentLabels,
+			instance.Labels,
```

Comment:
good catch, otherwise the `deepequal` wound never run the update.

Reply:
seems we have that everywhere where we call the func?

* https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/controllers/openstackbaremetalset_controller.go#L264
* https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/controllers/openstackclient_controller.go#L162
* https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/controllers/openstackcontrolplane_controller.go#L916
* https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/controllers/openstackvmset_controller.go#L269

Reply:
Oh, I didn't realize that.  Yes, I think we should fix it in every location.  Should I update this PR?